server/src/main/java/org/candlepin/util/Util.java

```diff
@@ -547,4 +550,24 @@ public static String getHostname() {
     public static OffsetDateTime toDateTime(Date date) {
         return date != null ? date.toInstant().atOffset(ZoneOffset.UTC) : null;
     }
+
+    /**
+     * Takes a map and splits it into a list of maps each with one item from the given map.
+     *
+     * @param map a map to be split
+     * @return a list of single item maps
+     */
+    public static List<Map<String, String>> split(Map<String, String> map) {
```

Comment:
Where exactly is this needed? I can't find where this is used in your changes?

Reply:
Done.